examples/pagination/run.js

```diff
@@ -0,0 +1,86 @@
+import { initConfigOptions, setUpClient, logError } from '../../shared/utils.js'
+import util from 'util'
+
+function getQueryData() {
+    return {
+        queryBody: `query getOrders($limit: Int, $sort: [String!], $where: String) {
+                        orders (limit: $limit, sort: $sort, where: $where) {
+                            results {
+                                id
+                            }
+                        }
+                    }`,
+        endpoint: 'orders',
+        variables: {
+            limit: 50
+        }
+    }
+}
+
+async function processOrders (ctpClient) {
+    await paginateWithGraphQl(ctpClient, getQueryData())
+    await paginateWithRest(ctpClient, getQueryData())
+    await paginateWithSdk(ctpClient, getQueryData())
```

Comment:
Don't understand why we need to go through 3 different ways of pagination for same set of data.

Reply:
Check the directory structure and the documentation. Here we provide 3 different examples how one could paginate over data in an efficient and performant way.